web/src/OverviewPane.tsx

```diff
@@ -19,48 +22,63 @@ let OverviewPaneRoot = styled.div`
   background-color: ${Color.grayDark};
 `
 
-let AllServicesDividerRoot = styled.div`
+let ServicesDividerRoot = styled.div`
   display: flex;
   width: 100%;
   align-items: center;
 `
 
-let AllServicesDashLeft = styled.div`
-  width: ${SizeUnit(0.5)};
-  height: 0;
-  border-bottom: 1px dashed ${Color.grayLight};
+let ServicesLabel = styled.div`
+  font: ${Font.sansSerif};
+  color: ${Color.blue};
+  margin: 12px;
 `
 
-let AllServicesDashRight = styled.div`
+let ServicesContainer = styled.div`
   flex-grow: 1;
-  height: 0;
-  border-bottom: 1px dashed ${Color.grayLight};
+  flex-shrink: 1;
+  overflow: auto;
 `
 
-let AllServicesLabel = styled.div`
-  font: ${Font.sansSerif};
-  color: ${Color.blue};
-  margin: 12px;
-`
+function PinnedServicesDivider() {
+  return (
+    <ServicesDividerRoot>
+      <GridDividerPinSvg style={{ marginLeft: "28px" }} />
+      <ServicesLabel>Favorite Resources</ServicesLabel>
```

Comment:
Let's consistently refer to these as "Pinned," like in the sidebar. "Fave" would be if we dared to use the cute lil 💕 metaphor...

Reply:
done!